_posts/2020-03-09-o-openid-connect-slow-kilka.md

```diff
@@ -0,0 +1,106 @@
+---
+layout:    post
+title:     "O OpenID Connect słów kilka"
+published: true
+date:      2020-03-09 08:00:00 +0100
+author:    mhoja
+tags:
+    - openid connect
+    - oauth 2
+    - keycloak
+    - uwierzytelnianie użytkownika
+---
+
+W nawiązaniu do mojego poprzedniego wpisu pt.:  
+[**"Keycloak - uwierzytelnianie i autoryzacja użytkownika w aplikacji Angular/Spring Boot"** (link](https://blog.consdata.tech/2020/02/01/keycloak-uwierzytelnianie-autoryzacja-springboot-angular.html)),  
```

Comment:
1. tu jest chyba coś nie teges z nawiasami
2. używajmy post_url

Reply:
1. Te nawiasy są celowe - opisem linku jest:
_"Keycloak - uwierzytelnianie i autoryzacja użytkownika w aplikacji Angular/Spring Boot" (link)_
2. post_url nie używałem nigdy, mogę spróbować
3. celowo pogrubiłem tytuł, żeby rzucał się w oczy
4. (link) dałem po to, żeby było widać że to jest klikalne - dodanie samego linku do tekstu nie za bardzo jest widoczne
5. wrzuciłem w osobną linię żeby się nie łamało, po odpaleniu jekylla dobrze to wygląda imo:
![image](https://user-images.githubusercontent.com/22873005/75859272-fbbe1600-5df9-11ea-91b3-9762c9ad14fd.png)

Reply:
A może ikonka zamiast tekstu :)?

🔗

https://www.compart.com/en/unicode/U+1F517

Reply:
fajny pomysł :)